### ISSUE TITLE
[Action] Create a pre-event checklist issue type

### ISSUE THREAD

**Issue by claire-fletcher** [cncf/tag-env-sustainability#449]
### Description

Create a pre-event checklist issue type for the wg comms events lead to work through. Ensuring all comms things are completed.

### Outcome

issue type to make tracking what we need to do easier!

### To-Do

- [ ] ...

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by claire-fletcher** [comment]
The current checklist details: https://github.com/orgs/cncf/projects/10/views/13?pane=issue&itemId=44366418&issue=cncf%7Ctag-env-sustainability%7C270

**Comment by vallss** [comment]
> The current checklist details: https://github.com/orgs/cncf/projects/10/views/13?pane=issue&itemId=44366418&issue=cncf%7Ctag-env-sustainability%7C270

@claire-fletcher since in the doc we have two different lists, one for "Generic" events and one for "Kubecon", do we prefer to use a different issue template for Kubecon or we can have two checklist sections in the same template? 
I would suggest to use the same template and add this information as a guideline (i.e. "If this is a Kubecon event, then use checklist ABC, if it's not, then use checklist DEF").

Let me know what you think about it!

**Comment by vallss** [comment]
Here's the PR with both checklists https://github.com/cncf/tag-env-sustainability/pull/587

**Comment by claire-fletcher** [comment]
I agree with your suggestion, I'll take a look at the PR then we can use this for the next event! And we can add anything from this that is missing from the existing eu kubecon ticket
